### DISCUSSION TITLE
27c322 adapter

### DISCUSSION THREAD

**Discussion by Chomemel** [sanni/cartreader#755] [Hardware]
Hi Sanni,
I plan to print boards for 27c322 programming for HW3. Do you have a list of the components (size, values, ref)? I found some values in the schemas but some are not written.

**Comment by sanni** [comment]
The 27c322 programming doesn't work correctly, the eprom loses its data after some time. You're better off using a TL866 programmer.

If you still want to proceed on your own you will find more information through archive.org: https://web.archive.org/web/20190125061335/https://github.com/sanni/cartreader/wiki/Flash--and-Eprom-Adapter
Most of the parts were taken from this step-up converter: https://www.ebay.com/itm/201919379310

**Comment by Chomemel** [comment]
You mean that the EPROM works for a bit and then the data vanishes? How long after it is programmed? hours? days?

I know a programmers like TL866, GQ-4X or else would be better, but I was hoping that I could avoid buying one if the the cart reader was able to program some EPROMs.

**Comment by sanni** [comment]
Setting/resetting margin mode is missing from the current 27C322 code. Margin mode is needed for long time data retention. I can't make a prediction on how long it will hold the data without. I just never finished writing the code and then lost interest.

**Comment by Chomemel** [comment]
Thanks for answer Sanni.
Too bad I don't know anything about coding. But no worries, I'll find another way to get my 27c322/160 programmed.
Thanks for everything you've done so far.
